### PR TITLE
relocated cloneEvent method, updated places where it is called and ad…

### PR DESCRIPTION
**Pull request by juwara0**
* **Added** unit tests for frost-toggle's _changeTarget()
* **Updated** location of cloneEvent()
* **Updated** frost-toggle.js and frost-radio-button.js to use new path for cloneEvent()

Closes: https://github.com/ciena-frost/ember-frost-core/issues/278

### CODE REVIEW

File: addon/utils/events.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
 import Ember from 'ember'
-const {
-  $,
-  deprecate
-} = Ember
+const {deprecate} = Ember
```
Comment (dafortin) [inline addon/utils/events.js:2]: What are you deprecating here?

File: addon/utils/utils.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember'
```
Comment (dafortin) [inline addon/utils/utils.js:1]: Should we find a better name than `utils`? I feel like having a utils class in a utils folder is a bit utility overkill :P

File: addon/utils/utils.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+const {$} = Ember
```
Comment (dafortin) [inline addon/utils/utils.js:2]: Are you planning to add tests for this util?

File: addon/utils/events.js (hunk 1 above)
Comment (juwara0) [reply addon/utils/events.js:2]: https://github.com/juwara0/ember-frost-core/blob/62cb57155987c6148c1b9ad44f82b6ee93c10b72/addon/utils/events.js#L33

Comment (dafortin) [reply addon/utils/events.js:2]: Oh it was missing. Got it :) 👍

File: addon/utils/utils.js (hunk 3 above)
Comment (dafortin) [reply addon/utils/utils.js:2]: I will add the tests in a future PR.

File: addon/utils/utils.js (hunk 2 above)
Comment (dafortin) [reply addon/utils/utils.js:1]: I think we can live with this after discussion.

File: addon/utils/utils.js (hunk 3 above)
Comment (juwara0) [reply addon/utils/utils.js:2]: @dafortin Agreed to write tests for this util method.

**Pull request merged**